handlers/organizerHandler.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express");
+const router = express.Router();
+const { db } = require("../firebase");
+
+router.get("/", (req, res) => {
+  const organizer = db.collection("Organizer");
+  organizer
+    .get()
+    .then((snapshot) => {
+      const data = snapshot.docs.map((doc) => ({
+        id: doc.id,
+        comittee: doc.get("comittee"),
+        poc_name: doc.get("poc_name"),
+        poc_email: doc.get("poc_email"),
+        poc_contact: doc.get("poc_contact"),
+      }));
+      console.log(data);
+      res.status(200);
+      res.send(data);
+    })
+    .catch((error) => {
+      console.error(error);
+      res.status(500);
+      res.send("Something went wrong, please try again");
+    });
+});
+
+router.post("/", (req, res) => {
+  let poc_name = req.body.poc_name;
+  let poc_contact = req.body.poc_contact;
+  let poc_email = req.body.poc_email;
+  let comittee = req.body.comittee;
+
+  let message = "";
+
+  if (
+    poc_name != "" &&
+    comittee != "" &&
+    poc_email != "" &&
+    poc_contact != null
```

Comment:
why poc_contact is compared with null ?

Reply:
coz number hai, "" this will treat it as a string

Reply:
but your variable will always collect string data type from request.. have you typecasted to any specific datatype?